lib/plaintime.ts

```diff
@@ -43,7 +43,7 @@ type TemporalTimeToStringOptions = {
 function TemporalTimeToString(
   time: Temporal.PlainTime,
   precision: ReturnType<typeof ES.ToSecondsStringPrecision>['precision'],
-  options: TemporalTimeToStringOptions = undefined
+  options: TemporalTimeToStringOptions | undefined = undefined
```

Comment:
Could be `options?: TemporalTimeToStringOptions`?

Reply:
That playground link won't load for me, unfortunately (it gets stuck on "Downloading TypeScript"...) but are you referring to the function length?

Reply:
Oh, weird. Never heard of that TS Playground symptom. Anyway, here's what's in there: 

```ts
// TS code:
function f1(x: string | undefined = undefined) {}
function f2(x?: string) {}

// JS code that's emitted by the transpiler
function f1(x = undefined) { }
function f2(x) { }
```

The `= undefined` is different between the two, so I assumed it was better to match the runtime output whenever possible.

Reply:
Probably inconsequential because the length of that function is not observable, but I can see that divergences might make it harder to debug.